Lib/asyncio/selector_events.py

```diff
@@ -246,8 +246,16 @@ def _accept_connection2(self, protocol_factory, conn, extra,
                 self.call_exception_handler(context)
 
     def _ensure_fd_no_transport(self, fd):
+        fileno = fd
+        if not isinstance(fileno, int):
+            try:
+                fileno = int(fileno.fileno())
+            except (AttributeError, TypeError, ValueError):
+                # This code matches `selectors._fileobj_to_fd` function.
```

Comment:
Please avoid backticks -- without them the comment is still pretty clean.

Reply:
Removed, but this is a very personal preference -- for me it was easier to read that comment with backticks/quotes.